tests/ignite/distributed/utils/test_serial.py

```diff
@@ -9,7 +9,6 @@ def test_no_distrib(capsys):
 
     from ignite.distributed.utils import _model
 
-    print("test_no_distrib : dist: ", dist.is_available())
```

Comment:
Could you please remove both prints and "from ignite.distributed.utils import _model" if it is not used below.